src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -659,4 +694,22 @@ public String run(final Matcher m) {
 		return s;
 	}
 	
+	
+	/**
+	 *
+	 * Checks if an object is empty or null then it will return the message provided
+	 * 
+	 * @param object the object to check
+	 * @param message the message to reutn of conitions are true
+	 * 
+	 */
+	public String nullCheck(@Nullable String object, @NonNull String message) {
+		return (object == null || object.isEmpty()) ? message : object; // Null check first otherwise NullPointerExpection is thrown
+	}
```

Comment:
```suggestion
	/**
	 * Checks if a string is empty or null then it will return the message provided
	 * 
	 * @param string the String to check
	 * @param message the String to return if either condition is true
	 */
	public String getNullOrEmptyDefault(@Nullable String string, String message) {
		return (string == null || string.isEmpty()) ? message : string; // Null check first otherwise NullPointerExpection is thrown
	}
```

Reply:
Skript has NonNullByDefault annotations, so you don't have to add NonNull annotations yourself, only Nullable